internal/provider/resource_consul_cluster_test.go

```diff
@@ -14,26 +15,84 @@ import (
 var consulClusterUniqueID = fmt.Sprintf("test-%s", time.Now().Format("200601021504"))
 var consulClusterHVNUniqueID = fmt.Sprintf("test-hvn-%s", time.Now().Format("200601021504"))
 
-var consulCluster = fmt.Sprintf(`
-resource "hcp_consul_cluster" "test" {
-	cluster_id         = "%s"
-	hvn_id             = hcp_hvn.test.hvn_id
-	tier               = "development"
-	min_consul_version = data.hcp_consul_versions.test.recommended
-}
-`, consulClusterUniqueID)
+var consulCluster = consulClusterConfig(consulClusterUniqueID, `
+	ip_allowlist {
+    	address = "172.25.16.0/24"
+    	description = "this is an IPV4 address"
+  	}
+`)
```

Comment:
Could it be worthwhile to include an IPV6 address in the test case objects? Or do we currently only support IPV4 addresses?

Reply:
Hmm looks like creation fails with IPV6...

Reply:
@JolisaBrownHashiCorp For now, IPV6 is disallowed. I've added a validation to inform the user if they try to use one: https://github.com/hashicorp/terraform-provider-hcp/pull/455/commits/75ddd6a385e9a81519aef8ff24fa58ff0630e537

Reply:
Awesome. @Achooo Thanks for following up! May I ask what your slack handle is? I'm getting a deletion error when attempting to run the test suite and would like to inbox you the logs to discuss.

Reply:
Sure thing, it's `@ashvitha`, I've sent you a DM :)